### DISCUSSION TITLE
Please review and release video of the "So you think you know git?"

### DISCUSSION THREAD

**Discussion by xnox** [gitbutlerapp/gitbutler#2773] [Q&A] [answered]
https://fosdem.org/2024/schedule/event/fosdem-2024-3611-so-you-think-you-know-git/

Was presented by @schacon however the video is still not published at https://video.fosdem.org/2024/k1105/ as it is pending review and acceptance by @schacon . Can you please ensure @schacon reviews and approves publication of the video recording?

**Comment by krlvi** [comment]
Hey @xnox! Thanks for bringing this up! We'll be able to fix that on Tuesday as Scott is away currently

**Comment by schacon** [comment, marked as answer]
So, I reviewed it, but the sound quality is incredibly bad. We had mic'd me separately and provided the better sound file to FOSDEM but they didn't update the video with it. It seems like the whole thing is semi-automated, it's difficult to get a person to help or fix it. Anyhow, we made our own cut from their footage and uploaded it ourselves, since we never got a response from them:

https://www.youtube.com/watch?v=aolI_Rz0ZqY&t=2463s
